bin/prod-style-server

```diff
@@ -9,7 +9,7 @@ exec > >(tee -a "setup.log") 2>&1
 
 echo "Starting server setup..."
 
-# Check for existence of required docker-compose file
+# Check for existence of required docker compose file
```

Comment:
I think find and replace betrayed you here

Reply:
```suggestion
# Check for existence of required docker-compose file
```